composer.json

```diff
@@ -94,19 +94,12 @@
 		],
 		"phpstan": [
 			"phpstan analyze --ansi --memory-limit=2G"
-		]
-	},
-	"archive": {
-		"exclude": [
-			"*.yml",
-			"!vendor/",
-			"plugin-build/",
-			"node_modules/",
-			"!.wordpress-org/",
-			"docs/",
-			"wp-graphql.zip",
-			"!build",
-			"packages"
+		],
+		"zip": [
```

Comment:
This should _probably_ run `composer install --no-dev` first. 

I just ran `composer zip` locally and it created a built plugin with all the vendor dependencies which would cause problems if I were to upload it to WordPress